src/test/java/test/DynamicGraphTest.java

```diff
@@ -67,20 +67,18 @@ public void test8() {
     dg.addEdge(0, b1, x, y);
     dg.addEdge(0, c1, x, y);
     assertFreeNodesEquals(dg, y, x);
-    dg.setStatus( dg.getFreeNodes(), Status.RUNNING);
```

Comment:
Big 👍 to this request.

I'm always nervous when the tests get modified since this can potentially introduce a regression (even if this seems justified in this case). If a change is introduced and all it does is add tests without modifying existing ones, then I'm much more comfortable approving it since I'm pretty sure it won't introduce regressions.